packages/atlas/src/providers/joystream/provider.tsx

```diff
@@ -35,6 +39,14 @@ export const JoystreamProvider: React.FC = ({ children }) => {
 
   const proxyCallback = useCallback(<T extends object>(callback: T) => proxy(callback), [])
 
+  useEffect(() => {
```

Comment:
Your `useEffect` doesn't provide a dependency array, so it will every time the provider renders

Reply:
Thanks! That's fixed